lambdas/services/document_manifest_service.py

```diff
@@ -71,21 +94,21 @@ def download_documents_to_be_zipped(self):
                 logger.error(
                     f"{document.get_file_key()} may reference missing file in s3 bucket {document.get_file_bucket()}"
                 )
-                raise ManifestDownloadException(
-                    f"Reference to {document.file_key} that doesn't exist in s3"
+                raise DocumentManifestServiceException(
+                    status_code=500,
+                    message=f"Reference to {document.file_key} that doesn't exist in s3",
                 )
 
     def upload_zip_file(self):
         logger.info("Creating zip from files")
-        temp_output_dir = tempfile.mkdtemp()
 
-        zip_file_path = os.path.join(temp_output_dir, self.zip_file_name)
+        zip_file_path = os.path.join(self.temp_output_dir, self.zip_file_name)
```

Comment:
Also this method is the only one that uses Dynamo Service - I feel that since the document_service below contains all the logic for getting the DynamoDB info for this use case (I'd have called this , we should maybe move the last line of this method into that service?